content/agent/agent_checks.md

```diff
@@ -256,189 +255,9 @@ class HelloCheck(AgentCheck):
 
 As you can see, the check interface is really simple and easy to get started
```

Comment:
Should we remove these two lines? Do they relate to the example you removed below?

Reply:
👌 nop.. just bad phrasing